keywords/CouchbaseServer.py

```diff
@@ -250,3 +267,212 @@ def get_server_docs_with_prefix(self, url, bucket, prefix):
                 found_ids.append(row["$1"]["id"])
 
         return found_ids
+
+    def _wait_for_rebalance_complete(self, url):
+        """
+        Polls couchbase server tasks endpoint for any running rebalances.
+        Exits when no rebalances are in running state
```

Comment:
Interesting. So far I have not seen this as a problem. Regardless, adding another check right now.